app/views/layouts/application.html.erb

```diff
@@ -10,6 +10,10 @@
     <%= link_to 'Queries', queries_path %>
   </li>
 
+  <li class='<%= controller_name.in?(%w[recommended_links]) ? "active" : nil %>'>
```

Comment:
are you expecting to add more options? otherwise it could be just `controller_name == 'recomended_links'`, what do you think?